packages/stencil/src/components/datacom-avataravatar/datacom-avataravatar.css

```diff
@@ -0,0 +1,43 @@
+:host {
+  display: block;
+}
+
+.avatar {
+  display: flex;
+  font-family: 'Montserrat';
+}
+
+.avatar-image img,
+.avatar-initial {
+  display: block;
+  height: 48px;
+  width: 48px;
+  border-radius: 50%;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  background: #0a1839;
+  color: #fff;
```

Comment:
Might be better to have this as a css-variable so others can use this color (white as a color is quite common). Maybe you can add a new color variable into light.css and use it from there?